pkgs/applications/editors/leo-editor/default.nix

```diff
@@ -1,20 +1,20 @@
-{ stdenv, python3Packages, fetchFromGitHub, makeWrapper, makeDesktopItem }:
+{ stdenv, python3Packages, pyqt56, fetchFromGitHub, makeWrapper, makeDesktopItem }:
 
```

Comment:
Overriding the Python package set here, replacing the `pyqt5` attribute would be the preferred solution.

Reply:
mitmproxy is a good example